hailjwt/hailjwt/hailjwt.py

```diff
@@ -25,14 +26,20 @@ def __init__(self, secret_key):
         JWTClient._verify_key_preqrequisites(secret_key)
         self.secret_key = secret_key
 
-    def decode(self, token):
+    def decode_bytes(self, token):
         return jwt.decode(
             token, self.secret_key, algorithms=[JWTClient.__ALGORITHM])
 
-    def encode(self, payload):
+    def decode(self, token):
```

Comment:
This represents a breaking change; could we instead keep `decode` instead of `decode_bytes` and call the new `decode` `decode_as_str`? Same for encode below

Reply:
The common case of flask and aio needs a string, so I'd prefer the succinct name supports that. Breaking changes don't matter much to internal libraries anyway since we can fix all the use sites in the same PR.

Reply:
my bad, I misunderstood some errors I saw about encodings. I've reverted all this